test_suite/tests/ui/fail_with_invalid_codec_attrs.stderr

```diff
@@ -25,10 +19,10 @@ error: proc-macro derive panicked
    = help: message: scale-info: Bad index in `#[codec(index = …)]`, see `parity-scale-codec` error: Error("expected literal")
 
 error: expected literal
-  --> $DIR/fail_with_invalid_codec_attrs.rs:24:25
+  --> $DIR/fail_with_invalid_codec_attrs.rs:18:21
```

Comment:
Indeed. It is a matter or order: recent rusts put the errors in the order of appearance in the file, so errors on line 18 come before errors on line 24.

Given that the order is more correct on newer rusts, can we bump CI to use a more recent version?

Reply:
I believe the CI automatically uses the latest nightly which includes the required components (`rustfmt`, `clippy`).  I see there was a new one yesterday `1.54.0-nightly (bacf770f2 2021-05-05)`, it might be picked up on a rerun.

Reply:
It did pick up the latest one as above, but the UI test still fails, so might need to update those UI tests using `1.54.0-nightly`

Reply:
CI and my rustc seems to be in agreement now. Ty.